README.md

```
@@ -1 +1,81 @@
+This project follows the [Conventional Commits](https://www.conventionalcommits.org/) specification for commit messages. Your commit messages should be structured as follows:
+
+```
+<type>(<scope>): <description>
+
+[optional body]
+
+[optional footer(s)]
+```
+
+Common types include:
+- **feat**: A new feature
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)